frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -1,51 +1,305 @@
+    const { billing } = useValues(billingLogic)
+    const supportResponseTimes = {
+        [AvailableFeature.EMAIL_SUPPORT]: '2-3 days',
+        [AvailableFeature.PRIORITY_SUPPORT]: '4-6 hours',
+    }
+
+    return (
+        <Section title="Email an engineer">
+            <div className="grid grid-cols-2 border rounded [&_>*]:px-2 [&_>*]:py-0.5 mb-4 bg-bg-light">
+                <div className="col-span-full flex justify-between border-b bg-bg-white py-1">
+                    <div>
+                        <strong>Avg support response times</strong>
+                    </div>
+                    <div>
+                        <Link to={urls.organizationBilling([ProductKey.PLATFORM_AND_SUPPORT])}>Explore options</Link>
+                    </div>
+                </div>
+                {billing?.products
+                    ?.find((product) => product.type == ProductKey.PLATFORM_AND_SUPPORT)
+                    ?.plans?.map((plan, i) => (
+                        <React.Fragment key={`support-panel-${plan.plan_key}`}>
+                            <div className={plan.current_plan ? 'font-bold' : undefined}>
+                                {i == 1 ? 'Pay-per-use' : plan.name}
+                                {plan.current_plan && (
+                                    <>
+                                        {' '}
+                                        <span className="font-normal opacity-60 text-sm">(your plan)</span>
+                                    </>
+                                )}
+                            </div>
+                            <div className={plan.current_plan ? 'font-bold' : undefined}>
+                                {plan.features.some((f) => f.key == AvailableFeature.PRIORITY_SUPPORT)
+                                    ? supportResponseTimes[AvailableFeature.PRIORITY_SUPPORT]
+                                    : plan.features.some((f) => f.key == AvailableFeature.EMAIL_SUPPORT)
+                                    ? supportResponseTimes[AvailableFeature.EMAIL_SUPPORT]
+                                    : 'Community support only'}
+                            </div>
+                        </React.Fragment>
+                    ))}
+            </div>
+            <SupportForm />
+            <LemonButton
+                form="support-modal-form"
+                htmlType="submit"
+                type="primary"
+                data-attr="submit"
+                fullWidth
+                center
+                className="mt-4"
+            >
+                Submit
+            </LemonButton>
+            <LemonButton
+                form="support-modal-form"
+                type="secondary"
+                onClick={onCancel}
+                fullWidth
+                center
+                className="mt-2"
+            >
+                Cancel
+            </LemonButton>
+        </Section>
+    )
+}
+
 export const SidePanelSupport = (): JSX.Element => {
```

Comment:
We need two Storybook stories here: 1. Free help, no form 2. Paid help, with form

Reply:
We already haves ones for all the other panels, so should be super simple to get done